File: AtCoder/ABC/154/python/hello.py
t = 0
for i in range(1, 314159 + 1):
    S = str(i)
    cnt = 0
    for s in S:
        if s != '0':
            cnt += 1
    if cnt == 3:
        print(S)
        t += 1
print(t)

File: AtCoder/ABC/198/cpp/hello.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

int main() {
  vector<ll> V;
  for (ll i = 0; i < 10; ++i) {
    V.push_back(i);
  }
  ll cnt = 0;
  do {
    // for (auto v : V) {
    //   printf("%lld ", v);
    // }
    // puts("");
    ++cnt;
  } while (next_permutation(V.begin(), V.end()));
  printf("%lld\n", cnt);
}